pkgs/by-name/py/pyprland/package.nix

```
@@ -0,0 +1,24 @@
+{ lib, fetchFromGitHub, python311 }:
+
+python311.pkgs.buildPythonApplication rec {
+  pname = "pyprland";
+  version = "1.4.1";
+  pyproject = true;
```

```suggestion
  format = "pyproject";
```

while your original version works, I would prefer the version that's more common/consistent in nixpkgs.